py/desitarget/mock/log.txt

```diff
@@ -0,0 +1,155 @@
+20:35:22
```

Comment:
Logs like this should not be checked into the repo. You'll need to scrub your commit history in order to remove this.

Reply:
I'm still trying to break some engrained bad habits, hence I work on my fork to start.  Happy to clean that up for the final version.